egs/librispeech/asr/simple_v1/mmi_att_transformer_decode.py

```diff
@@ -285,7 +285,8 @@ def main():
             num_classes=len(phone_ids) + 1,  # +1 for the blank symbol
             subsampling_factor=4,
             num_decoder_layers=num_decoder_layers,
-            vgg_frontend=True)
+            vgg_frontend=True,
+            is_espnet_structure=True)
```

Comment:
Should have this in training script too

Reply:
.. and it's better if you change the directory name, when changing the model structure.
you can remove a couple of older components of the filename, to stop it getting too long.

Reply:
>Should have this in training script too

added.
> .. and it's better if you change the directory name, when changing the model structure.
you can remove a couple of older components of the filename, to stop it getting too long.
```
-    -noam-mmi-att-musan-sa-vgg
+    -mmi-att-sa-vgg-normlayer
```